### ISSUE TITLE
360° Support

### ISSUE THREAD

**Issue by LeoNatan** [efremidze/Shiny#4]
Cool idea!

When looking at Apple’s implementation, the layer (or whatever they are using) is wrapping around, so it goes from gray back to color, whereas in the implementation here, once the layer is offset enough, the view will remain gray even as the phone is tilted more and more.

**Comment by efremidze** [comment]
Thanks, I think they offset the motion and flip the gradient: [UIColor.gray, UIColor.red, UIColor.green, UIColor.blue]. What do you think?

**Comment by efremidze** [comment]
I'm going to try to use a spherical gradient or replicator layer to replicate the behavior.

**Comment by LeoNatan** [comment]
Sorry I missed your previous comment. Both approaches could work. With a spherical gradient, you just have to size the layer appropriately for 360° and offset it appropriately.

**Comment by umarF** [comment]
Yes please add 360° gradient, it will really help in case of holding device vertically in hand. Right now it just shows grey color while holding device vertically and moving.

**Comment by efremidze** [comment]
Working on the 360 support in the [develop branch](https://github.com/efremidze/Shiny/tree/develop) if anyone wants to help.

**Comment by efremidze** [comment]
I've added 360° support but I locked it to the vertical axis. Please take a look.

**Comment by umarF** [comment]
Yes this is much more practical and suitable !
Awesome.
